wallet/wallet.go

```diff
@@ -498,6 +498,11 @@ out:
 				continue
 			}
 			timeout = req.lockAfter
+			if timeout == nil {
+				log.Info("Wallet unlocked without timeout")
+			} else {
+				log.Info("Wallet temporarily unlocked")
```

Comment:
Hmmm well `dcrwallet` does the print here: https://github.com/decred/dcrwallet/blob/master/rpc/legacyrpc/methods.go#L3489-L3496

I'd really prefer it to print how long it's being unlocked for like in `dcrwallet`.  That, in my opinion, is much more useful information than just saying temporarily.